ansible/site.yml

```diff
@@ -0,0 +1,22 @@
+- hosts: bootstrap
+  remote_user: ubuntu
+  become: yes
+  become_method: sudo
+  roles:
+    - bootstrap
```

Comment:
There is always only one host with the bootstrap role, and other `masters` are formed by marking them as such with labels and simply scaling replicas or I am misunderstanding?